public/content/en/basics/associative-arrays.md

```diff
@@ -1,36 +1,64 @@
 # Associative Arrays
 
 D has built-in *associative arrays* also known as hash maps.
+An associative array is composed of `key`, `value` pairs
+for which a unique `key` allows access to its associated value.
```

Comment:
I think it's a good style if the first paragraph is a short intro to the topic, s.t. everyone is on the same page - we should probably check other pages for this too.